src/main/java/com/epam/paymentsystem/service/repository/impl/BankAccountRepositoryImpl.java

```diff
@@ -0,0 +1,48 @@
+package com.epam.paymentsystem.service.repository.impl;
+
+import com.epam.paymentsystem.service.model.BankAccount;
+import com.epam.paymentsystem.service.model.enums.Status;
+import com.epam.paymentsystem.service.repository.BankAccountRepository;
+import org.springframework.stereotype.Component;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Component
```

Comment:
Use @Repository annotation instead of @Component, it has actually no difference, but first one has more readability to reviewers